packages/eslint-plugin/src/rules/strict-boolean-expressions.ts

```diff
@@ -715,11 +715,16 @@ export default util.createRule<Options, MessageId>({
             context.report({
               node,
               messageId: 'conditionErrorNullableObject',
```

Comment:
[Bug] So I just scrolled up a bit in the file and found another usage of `messageId: 'conditionErrorNullableObject'`. Looks like there's another `fix` that uses this same message ID. I think this'll need to also switch that one to a `suggestion`.